cfgov/jinja2/v1/_includes/molecules/hero.html

```diff
@@ -27,36 +27,36 @@
    value.background_color: A string containing the color code (in any format)
                            to set as the hero background color.
 
-   value.is_white_text:    A boolean to change the text color to white.
+   value.photo:            A boolean to change the hero to an overlay hero.
 
-   value.is_bleeding:      A boolean that indicates that the image bleeds
-                           vertically.
+   value.white_text:       A boolean to change the text color to white.
 
-   value.is_overlay:       A boolean to change the hero to an overlay hero.
+   value.bleed:            A boolean that indicates that the image bleeds
+                           vertically.
 
    ========================================================================== #}
 
 
 {# TODO: This is simple now, but in the future we'll hopefully be able to
    provide 1x and 2x values as well. We should update this to be an image
    array delivered by Django in the future. #}
-{% set img = image(value.image, 'original') %}
+{% set img = image(value.large_image, 'original') %}
 
 {% if value.small_image %}
     {% set sm_img = image(value.small_image, 'original') %}
 {% else %}
-    {% set sm_img = image(value.image, 'original') %}
+    {% set sm_img = image(value.large_image, 'original') %}
```

Comment:
Looks like `sm_img` (small image) is being set to `large_image`.

Reply:
Hm. So perhaps the `sm_img` variable is poorly named. Renaming it seems out of scope for this PR.

Reply:
This is just the fallback if a separate small image is not chosen in Wagtail. OK as-is.